etcd_bootstrap_script.sh

```diff
@@ -22,70 +21,66 @@ trap_and_propagate() {
 }
 
 start_managed_etcd(){
-      arch=$(uname -m)
-      if [ $arch = "aarch64" ] || [ $arch = "arm64" ]; then
-          # Running etcd on ARM has experimental support for version 3.4.x
-          # https://etcd.io/docs/v3.4/op-guide/supported-platform/
-          export ETCD_UNSUPPORTED_ARCH=arm64
-      fi
-      rm -rf $VALIDATION_MARKER
-      CONFIG_FILE=/etc/etcd.conf.yaml
-      curl --retry 5 --retry-delay 5 --retry-all-errors "$BACKUP_ENDPOINT/config" -o $CONFIG_FILE
-      minimumsize=50
-      actualsize=$(wc -c <$CONFIG_FILE)
-      if [ $actualsize -le $minimumsize ]; then
-          echo "downloaded config file size is less than $(minimumsize) bytes"
-          exit 1
-      fi
-      etcd --config-file $CONFIG_FILE &
-      ETCDPID=$!
-      trap_and_propagate $ETCDPID INT TERM
-      wait $ETCDPID
-      RET=$?
-      echo $RET > $VALIDATION_MARKER
-      exit $RET
+    arch=$(uname -m)
+    if [ $arch = "aarch64" ] || [ $arch = "arm64" ]; then
```

Comment:
Can you also enclose $arch in double quotes? This is usually done to prevent gobbling and word splitting.